controller/model/session_manager_test.go

```diff
@@ -24,7 +24,7 @@ func (ctx *TestContext) testSessionIdempotency(t *testing.T) {
 	service := ctx.requireNewService()
 	service2 := ctx.requireNewService()
 	service.RoleAttributes = []string{eid.New()}
-	ctx.NoError(ctx.managers.EdgeService.Update(service, nil, change.New().SetSource("test")))
+	ctx.NoError(ctx.managers.EdgeService.Update(service, nil, change.New()))
```

Comment:
No "test"?

Reply:
I decided providing entity change data for tests, unless entity change events were being tested, was of little value and just extra maintenance overhead. I decided this the third time I changed the API and had to go through and update all the change usages again.